clowder-framework/clowder2-frontend
# When authorization token expires need to logout user and then redirect to /login page #43

This PR mostly address the issue when token is invalid or expired in local storage, frontend will automatically log user out.
To test:
- find "Authorization" in your local storage, and modify the "bearer token" to anything that doesn't make sense
- try different actions on frontend. Whenever you see a 401 error in the console, you should also see it automatically redirect user to the login page

A few other things addressed in the PR:
- make a error reducer and a common function to handle errors to make the code cleaner
- fix a minor bug on Login component. When user already logged in, it will redirect to home page
